addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,32 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
```

Comment:
Not really an issue here, but we could consider how to change it for master where we think more about a signature like _format_vat(self, country_id, type='vat')

Reply:
as the format depends on identification type, by now we need there all the values. Let's think it..